commands/rootCmd.go

```diff
@@ -0,0 +1,83 @@
+/*
+Package commands contains github-audit shell commands related logic.
```

Comment:
why file name is rootCmd.go and not github-audit.go or command.go. Or can u rename main.go to github-audit.go by the name of cli.

Reply:
changed